arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/common/CommonFragment.java

```diff
@@ -125,24 +125,6 @@ public Trace trace(Trace trace) {
         .counterNsr((short) nonStackRowsCounter);
   }
 
-  private Bytes gasCostToTrace() {
```

Comment:
This was moved to `CommonFragmentValues`. The computation takes place using only values from there.